tuf/ngclient/updater.py

```diff
@@ -368,210 +372,80 @@ def _load_targets(self, role: str, parent_role: str) -> None:
             self._trusted_set.update_delegated_targets(data, role, parent_role)
             self._persist_metadata(role, data)
 
-    def _preorder_depth_first_walk(self, target_filepath) -> Dict:
+    def _preorder_depth_first_walk(
+        self, target_filepath: str
+    ) -> Union[Dict[str, Any], None]:
```

Comment:
```suggestion
    ) -> Optional[Dict[str, Any]]:
```

these mean union is then likely unused import